scene/gui/popup_menu.cpp

```diff
@@ -273,7 +273,15 @@ void PopupMenu::_parent_focused() {
 			window_parent = Object::cast_to<Window>(window_parent->get_parent()->get_viewport());
 		}
 
-		Rect2 safe_area = DisplayServer::get_singleton()->window_get_popup_safe_rect(get_window_id());
+		Rect2 safe_area;
+		DisplayServer::WindowID window_id = get_window_id();
```

Comment:
I'd suggest naming this `p_window` to not shadow the existing private variable. It would also reflect the name of the `window_get_popup_safe_rect()` argument.

Reply:
Note that `p_` is a prefix we use for function/method parameters, so this wouldn't be a good name for a locally declared variable.

The simplest here might just be not to create a local variable, it's fine to call `get_window_id()` twice, it's a fairly trivial getter.

Reply:
Whenever there is conflict, I just use a shortened name. In this case it would be something like `wid`.

Reply:
That slipped out! Thanks, I renamed it to `wid` as suggested by @KoBeWi to avoid conflict